lib/bolt/cli.rb

```diff
@@ -483,6 +494,19 @@ def outputter
       @outputter ||= Bolt::Outputter.for_format(@config[:format])
     end
 
+    def list_tasks
+      Puppet.initialize_settings
+      Puppet::Pal.in_tmp_environment('bolt', modulepath: [BOLTLIB_PATH] + @config[:modulepath], facts: {}) do |pal|
```

Comment:
Can we make an in_bolt_compiler function so we don't forget to update this.

Reply:
What do you mean forget to update this?

Reply:
Added `in_bolt_compiler`.